explore-analyze/elastic-inference.md

```diff
@@ -7,7 +7,13 @@ navigation_title: Elastic Inference
 
 # Elastic {{infer-cap}}
 
-There are several ways to perform {{infer}} in the {{stack}}. This page provides a brief overview of the different methods:
+## Overview
 
+{{infer-cap}} is a process of using a {{ml}} trained model to make predictions or operations - such as text embedding, or reranking - on your data.
+You can use {{infer}} during ingest time (for example, to create embeddings from textual data you ingest) or search time (to perform [semantic search](/solutions/search/semantic-search.md)).
+There are several ways to perform {{infer}} in the {{stack}}:
+
+* [Using the Elastic {{infer-cap}} Service](elastic-inference/eis.md)
+* [Using `semantic_text` if you want to perform semantic search](/solutions/search/semantic-search/semantic-search-semantic-text.md)
```

Comment:
This enumeration is a list of things of different types:
- Elastic provides two "backends" for inference: EIS and ML nodes (trained models in the cluster)
- Both can be access via the Inference API. The Inference API also enables access to 3rd party inference "backends"
- `semantic_text` is an index mapping level interface that uses the Inference API.

Not necessarily saying we need to get into this detail here, but the things in this list are not equal, they build on top of each other.